static/app/components/timeline/index.tsx

```diff
@@ -0,0 +1,177 @@
+import {useMemo} from 'react';
+import styled from '@emotion/styled';
+
+import {getFormattedTimestamp} from 'sentry/components/events/interfaces/breadcrumbs/breadcrumb/time/utils';
+import {Tooltip} from 'sentry/components/tooltip';
+import {space} from 'sentry/styles/space';
+import {defined} from 'sentry/utils';
+import type {Color} from 'sentry/utils/theme';
+
+export interface ColorConfig {
+  primary: Color;
+  secondary: Color;
+}
+
+export interface ItemProps {
```

Comment:
make sure the things being exported have clear names even if it feels repetitive
```suggestion
export interface TimelineItemProps {
```